High score Javascript MOSTLY finished.

nicholas4418 opened the pull request:
Needs testing /Need to change prompt to modal

nicholas4418 committed:
High score Javascript MOSTLY finished. Needs testing /Need to change prompt to modal

jnollen23 commented on assets/js/highscore.js:
```diff
@@ -0,0 +1,60 @@
+
+//The purpose of this function is to check if the player's score is a "high score"
+//The funciton should be called by whatever script will control the "game over" part of the games
+function highScoreChecker(gameScore) {
```
Need to have a way to pass  the game name when a game calls the highscorechecker function

nicholas4418 committed:
Update highscore.js

nicholas4418 committed:
Added getUser() function

getUser() added so that saveHighScore() in the highscore.js can access the username.

nicholas4418 committed:
Altered saveHighScore() function / removed prompt

removed the "prompt" code and set "playername" to the getUser() function located in the CurrencyStorage.js

nicholas4418 committed:
Removed unnecessary leftover code

nicholas4418 committed:
Altered highScoreChecker()

Removed calling displayHighScores() from highScoreChecker(). Also added "var highScoresArray" to displayHighScores()
